src/middleware/genesys/README.md

```diff
@@ -1,34 +1,72 @@
+
+1. Update [`purecloudSetup.ts`](../../serviceDesks/genesys/purecloudSetup.ts) to populate it with your information.
+
+  - `ORGANIZATION_ID`: Your Genesys Cloud organization ID. You can find this ID in the [Genesys Cloud settings](https://help.mypurecloud.com/faq/how-do-i-find-my-organization-id/).
+  - `DEPLOYMENT_ID`: The ID of the widget you created in the previous step.
+
+1. Go to the project root directory and edit the `.env` file. Update the `SERVICE_DESK_CLASS` variable to `GenesysServiceDesk`.
+
+1. From the project root directory, run `npm run dev`. If you've linked everything to your Genesys Cloud account correctly, you should be able to connect to an agent in Genesys Cloud.
+
+To configure advanced functionality such as authenticated chat and agent availability status, follow these steps:
+
+1. If you haven't done so already, [set up a Genesys Cloud OAuth client](https://help.mypurecloud.com/articles/create-an-oauth-client/).
+
+1. In the `example-server` subdirectory, rename or copy `.env-sample` to `.env`.
+
+1. In the `.env` file, update the values of the `GENESYS_CLIENT_ID` and `GENESYS_CLIENT_SECRET` to the credentials from your Genesys OAuth client.
+
+1. From the `example-server` directory, run `npm i`.
+
+1. From the `example-server` directory, run `npm start`. This starts a server on port 3000 of your local machine.
+
+1. The server needs to be accessed from the browser of your end users. If you do not have your own hosted environment and you wish to expose your local development for testing, consider using a service such as [ngrok](https://ngrok.com/) to create a public URL:
+
+  ```
+  ngrok http http://localhost:3000`
+  ```
+
+1. In [`purecloudSetup.ts`](../../serviceDesks/genesys/purecloudSetup.ts), set the value of `AUTH_SERVER_BASE_URL` to the public URL for your server. Make sure you specify the `https://` URL.
+
+1. In the Genesys Cloud user interface, open the settings for your web chat widget and enable **Require Authentication**.
+
+1. In the **Authentication URL** field, specify `https://<server-url>/jwt`, where `<server-url>` is the public URL for your server. Click **Save**.
+
+1. In [`genesysServiceDesk.ts`](../../serviceDesks/genesys/genesysServiceDesk.ts), set the `WIDGET_REQUIRES_AUTHENTICATION` and `AUTHENTICATED_CALLS_ENABLED` flags to `true`.
+
+  **Note:** These flags are independent, so you do not have to enable both simultaneously. For example, if you set `WIDGET_REQUIRES_AUTHENTICATION = false` and `AUTHENTICATED_CALLS_ENABLED = true`, authentication willl be disabled in the web chat widget, but the server will still use an OAuth token to authenticate when accessing agent availability status.
+
+1. From the project root directory, run `npm run dev` again, and test. The server should now authenticate the chat and fetch agent availability.
```

Comment:
numbering incorrect, due to the Note, `From the project root..` is set to 1

Reply:
added indentation to get correct numbering